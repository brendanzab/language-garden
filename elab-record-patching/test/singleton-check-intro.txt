let _ : fun (A : Type) (x : A) -> A [= x] :=
  fun A x => x;
  --         ^ Elaborates to `#sing-intro x`
  --           when coercing from `A` to `A [= x]`

Type
