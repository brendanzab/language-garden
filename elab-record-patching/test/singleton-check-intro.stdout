<stdin> : Type :=
  let _ : fun (A : Type) (x : A) -> A [= x] :=
    fun (A : Type) (x : A) => #sing-intro;
  Type
